ILRepack.Tests/ILRepack.Tests.csproj

```diff
@@ -40,22 +40,8 @@
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="BamlParser, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\BamlParser.1.0.0\lib\net35\BamlParser.dll</HintPath>
-    </Reference>
-    <Reference Include="Fasterflect, Version=2.1.3.0, Culture=neutral, PublicKeyToken=38d18473284c1ca7, processorArchitecture=MSIL">
-      <HintPath>..\packages\fasterflect.2.1.3\lib\net40\Fasterflect.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="Moq">
-      <HintPath>..\packages\Moq.4.2.1409.1722\lib\net40\Moq.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.framework">
-      <HintPath>..\packages\NUnit.2.6.4\lib\nunit.framework.dll</HintPath>
-    </Reference>
-    <Reference Include="PresentationCore" Condition=" '$(Configuration)' != 'Debug_unix' " />
```

Comment:
"Debug_unix" is not needed anymore ?

Reply:
oh removed when re-adding the reference as it was failing to compile in Visual Studio
I think it is not building if it is under unix anyway, unless you ignore all the source related to UI
imo it's better to split into a separate project if we want to have the test running in *nix env

Reply:
I think these were actually needed so ilrepack could be compiled on linux

Reply:
I'm not sure if there is actually people using it under Linux, for instance, but I guess it would be nice to keep it or have it working.
Do you want to do another change after this one ?

Reply:
ok I know why, by default VS is using Debug_unix as configuration and it fails the build, adding it back

Reply:
https://github.com/gluck/il-repack/pull/102 I did this a while ago.

I remember we said that Linux support is nice to have if we can easily keep it.

In this case, I think we can keep the condition even in the new format, no?

Reply:
yes, pushed the change that keeps it